### ISSUE TITLE
Cannot export IOS app with single bundle ID 

### ISSUE THREAD

**Issue by stonekase** [godotengine/godot#34302]
**Godot version:**
v3.1.2
**OS/device including version:**
Mac OSX 10.13
**Issue description:**
Cannot export ios app with a single bundle Identifier Example : "whotfire".
I have to manually export it with a fake bundle_ID EG:"com.whotfire" just to export 
the change it in Xcode. this should be a simple fix.
<img width="514" alt="Screenshot 2019-12-12 at 11 50 57" src="https://user-images.githubusercontent.com/5037778/70706164-b34cfb80-1cd5-11ea-878e-88177842d81d.png">

**Steps to reproduce:**
just input a single bundle Identifier
**Minimal reproduction project:**
just try export app with single bundle identifier

<img width="967" alt="Screenshot 2019-12-12 at 11 42 27" src="https://user-images.githubusercontent.com/5037778/70706006-5ea98080-1cd5-11ea-95d2-486bdddac95f.png">

**Comment by stonekase** [comment]
@Calinou  we selecting what to respond to now?

**Comment by Calinou** [comment]
If Xcode recommends this for application deployment, it makes sense for Godot to require it. To make this more discoverable, we can provide an example in the error message, like this:

```text
Invalid Identifier: The Identifier must have at least one "." separator. Example: "com.example.my_project"
```

**Comment by stonekase** [comment]
> If Xcode recommends this for real-world application deployment, it makes sense for Godot to require it. To make this more discoverable, we can provide an example in the error message like this:
> 
> ```
> Invalid Identifier: The Identifier must have at least one "." separator. Example: "com.example.my-project"
> ```

but that is not the case as you can see from my xcode project. Secondly, i am actually porting my game from defold to godot and defold doesnt have that problem since the bundle ID was created and operating long before Godot.
So should games not be ported to Godot , just because Godot cannot handle non-reverse domain?

**Comment by stonekase** [comment]
<img width="1468" alt="Screenshot 2019-12-13 at 19 24 41" src="https://user-images.githubusercontent.com/5037778/70822636-4ffac080-1dde-11ea-86e3-1f180a7a7cd1.png">

Just for clarity, this is my xcode project with no errors or warning about reverse domain.

**Comment by NoFr1ends** [comment]
We should consider to show a warning instead.

**Comment by stonekase** [comment]
> We should consider to show a warning instead. It is allowed to upload these apps to the App Store, as long as the Bundle ID is unique, I don’t really see a reason to prevent the user from doing so.

I will really appreciate that instead of creating a fake bundle ID "com.whotfire" in Godot just to change it manually to "whotfire" in xcode everytime i export

**Comment by akien-mga** [comment]
CC @volzhs @bruvzg

**Comment by bruvzg** [comment]
Apple recommends reverse domain name notation, but it is not enforced. ID check code was copy pasted from Android as is (where it is enforced) alongside with the checks for other required properties.

Using "com.example.game" placeholder and check for empty string should be OK.

**Comment by bruvzg** [comment]
Fixed in #34339
